docs/ref/models/fields.txt

```diff
@@ -319,6 +319,41 @@ characters that aren't allowed in Python variable names -- notably, the
 hyphen -- that's OK. Django quotes column and table names behind the
 scenes.
 
+
+``db_comment``
```

Comment:
I think this whole section should be moved underneath the others, I think the others are more important and things should generally be added at the bottom.

Reply:
The list is alphabetized. This should remain in its current location so that the list remains alphabetized.